README.md

```diff
@@ -1,149 +1,158 @@
-
-For more information go to https://openjfx.io.
-
-## HelloFX
-
-Contains samples of a simple HelloFX class that can be run from command line, with or without build tools.
-
-### CLI
-
-`hellofx` [sample](HelloFX/CLI): simple HelloFX class to run on command line, without build tools
-
-### Maven
-
-`hellofx` [sample](HelloFX/Maven): simple HelloFX class to run with Maven
-
-### Gradle
-
-`hellofx` [sample](HelloFX/Gradle): simple HelloFX class to run with Gradle
-
-## CommandLine
 
-Contains samples of modular and non-modular projects that can be run from command line, with or without build tools.
+Description
+---
 
-### Non-Modular
+This repository contains a collection of HelloFX samples. Each one is a very simple 
+HelloWorld sample created with JavaFX 11 that can be run with different options and build tools.
 
-#### CLI
-
-`hellofx` [sample](CommandLine/Non-modular/CLI): non-modular project to run on command line, without build tools
-
-#### Maven
-
-`hellofx` [sample](CommandLine/Non-modular/Maven): non-modular project to run with Maven
-
-#### Gradle
-
-`hellofx` [sample](CommandLine/Non-modular/Gradle): non-modular project to run with Gradle
-
-### Modular
-
-#### CLI
-
-`hellofx` [sample](CommandLine/Modular/CLI): modular project to run on command line, without build tools
-
-#### Maven
-
-`hellofx` [sample](CommandLine/Modular/Maven): modular project to run with Maven
+The related documentation for each sample can be found [here](https://openjfx.io/openjfx-docs/).
 
-#### Gradle
+For more information go to https://openjfx.io.
 
-`hellofx` [sample](CommandLine/Modular/Gradle): modular project to run with Gradle
 
-## IDEs
 
-Contains samples of modular and non-modular projects that can be run from an IDE, with or without build tools.
+Content
+---
 
-### IntelliJ
+* [HelloFX samples](#HelloFX-Samples)
+* [CommandLine](#CommandLine)
+    - [_Non-modular samples_](#CLI-Non-Modular-Samples)
+    - [_Modular samples_](#CLI-Modular-Samples)
+* [IDEs](#IDEs)
+    - [IntelliJ](#IntelliJ)  
+      [_Non-modular samples_](#IntelliJ-Non-Modular-Samples)  
+      [_Modular samples_](#IntelliJ-Modular-Samples)  
+    - [NetBeans](#NetBeans)  
+      [_Non-modular samples_](#NetBeans-Non-Modular-Samples)   
+      [_Modular samples_](#NetBeans-Modular-Samples) 
+    - [Eclipse](#Eclipse)  
+      [_Non-modular samples_](#Eclipse-Non-Modular-Samples)   
+      [_Modular samples_](#Eclipse-Modular-Samples)
+* [License](#License)
+* [Contribution](#Contribution)
 
-#### Non-Modular
 
-##### Java
 
-`hellofx` [sample](IDE/IntelliJ/Non-Modular/Java): non-modular project to run from IntelliJ
+HelloFX samples<a name="HelloFX-Samples" />
+---
 
-##### Maven
+Contains samples of a simple HelloFX class that can be run from command line, with 
+or without build tools.
 
-`hellofx` [sample](IDE/IntelliJ/Non-Modular/Maven): non-modular project to run from IntelliJ, with Maven
+Build Tools | Sample | Description
```

Comment:
It's always one build tool, so maybe just use the singular form.

Reply:
This was something about which I think also, but I wasn't sure. Updated :).